src/packer.pkr.hcl

```
@@ -76,18 +76,18 @@ source "amazon-ebs" "example" {
   region             = var.build_region
   region_kms_key_ids = var.region_kms_keys
   skip_create_ami    = var.skip_create_ami
-  source_ami         = data.amazon-ami.debian_bullseye.id
-  ssh_username       = "admin"
+  source_ami         = data.amazon-ami.ubuntu_server_jammy.id
+  ssh_username       = "ubuntu"
   subnet_filter {
     filters = {
       "tag:Name" = "AMI Build"
     }
   }
   tags = {
-    Application        = "Example"
-    Base_AMI_Name      = data.amazon-ami.debian_bullseye.name
+    Application        = "Ubuntu Server"
+    Base_AMI_Name      = data.amazon-ami.ubuntu_server_jammy.name
     GitHub_Release_URL = var.release_url
-    OS_Version         = "Debian Bullseye"
+    OS_Version         = "Ubuntu Jammy"
```

I think we should use the full version information for the tag.

```suggestion
    OS_Version         = "Ubuntu Jammy Jellyfish"
```